src/mixxx.cpp

```diff
@@ -598,6 +596,11 @@ void MixxxMainWindow::initializeWindow() {
     Pal.setColor(QPalette::Background, MenuBarBackground);
     m_pMenuBar->setPalette(Pal);
 
+    // restore window size/position
+    QSettings settings("mixxxdj", "mixxx");
```

Comment:
We don't use QSettings for storing user preferences (which would write to another configuration file independent of mixxx.cfg). You should use UserSettings instead -- which you can get here via ```m_pSettingsManager->settings()```.